plugins/BEdita/Core/src/Model/Action/BaseAction.php

```diff
@@ -72,4 +73,23 @@ protected function initialize(array $config)
     {
         return $this->execute($data);
     }
+
+    /**
+     * Allowed object `status` condition
+     *
+     * @return array Empty array if all `status` are allowed otherwise a list of allowed values
+     */
+    protected function statusCondition()
```

Comment:
I'm not sure this is the right place for that piece of logic. `BaseAction` is intended as base class for all actions not necessarily for BEdita Objects action, so  I think is preferable to move elsewhere, maybe in a Trait.

Reply:
I wasn't sure about that either.
Could be a base class, but probably an `ObjectConditionsTrait` is better.